v3/util/time.go

```diff
@@ -63,6 +63,7 @@ var (
 	CABFBRs_1_6_9_Date          = time.Date(2020, time.March, 27, 0, 0, 0, 0, time.UTC)
 	CABFBRs_1_7_1_Date          = time.Date(2020, time.August, 20, 0, 0, 0, 0, time.UTC)
 	AppleReducedLifetimeDate    = time.Date(2020, time.September, 1, 0, 0, 0, 0, time.UTC)
+	CABFBRs_SC48_EffectiveDate  = time.Date(2021, time.August, 21, 0, 0, 0, 0, time.UTC) //TODO: get the right name for this
```

Comment:
It should go out via e-mail today that this is CABFBRs_1_8_0_Date - 1.7.9 was SC47, and SC48 finished review on Saturday

Reply:
That would explain why I couldn't find out what term I should be using if it hadn't been said anywhere yet.  Thanks!